src/Aspire.Hosting.Azure.ServiceBus/AzureServiceBusEmulatorResource.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+
+namespace Aspire.Hosting.Azure;
+
+/// <summary>
+/// Wraps an <see cref="AzureServiceBusResource" /> in a type that exposes container extension methods.
+/// </summary>
+/// <param name="innerResource">The inner resource used to store annotations.</param>
+public class AzureServiceBusEmulatorResource(AzureServiceBusResource innerResource) : ContainerResource(innerResource.Name), IResource
+{
+    // The path to the emulator configuration file in the container.
+    internal const string EmulatorConfigJsonPath = "/ServiceBus_Emulator/ConfigFiles/Config.json";
```

Comment:
assume this is still WIP as would need a way to supply config.json based ont he docs as well?

Reply:
This is the internal path for the emulator, this won't change. We do bind your own files to this internal path. Use the `WithConfigJson(yourpath)` for that. And internally it will bind to this path.

Reply:
I will add another method to configure a `JsonObject` representing the document in case we want to alter either the generated one of the bound one.

Reply:
> This is the internal path for the emulator, this won't change. We do bind your own files to this internal path. Use the `WithConfigJson(yourpath)` for that. And internally it will bind to this path.

Oh, that would be nice to get in for Event Hubs emulator to unblock a few things while the API evolves to match this app model idea.

Reply:
Done, added `ConfigureJson` in the `RunEmulator`'s lambda.